src/molecule/test/functional/test_command.py

```diff
@@ -287,6 +296,20 @@ def test_command_test(scenario_to_test, with_scenario, scenario_name, driver_nam
     run_test(driver_name, scenario_name)
 
 
+@pytest.mark.serial
+@pytest.mark.parametrize(
+    "scenario_to_test, driver_name, scenario_name, platform_name",
```

Comment:
Let's use distinct values for readability:
```suggestion
    ("scenario_to_test", "driver_name", "scenario_name", "platform_name"),
```

Reply:
The value has been changed to a tuple.

But I noticed a string separated by comma is used in all other functions as well as [parametrize](https://docs.pytest.org/en/6.2.x/parametrize.html)

Could you give us some more details on this difference?